### PR TITLE
Add a list of numbers

### LINKED ISSUE #7: Handle simple strings

### PR DESCRIPTION
**Pull request by THume**
Fixes #7

### CODE REVIEW

**Comment by THume** [comment]
@mikepiehler please review

File: src/StringCalc/StringCalc.Core/StringCalculator.cs [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+﻿using System;
+
+namespace StringCalc.Core
 {
     public class StringCalculator
     {
         public int Add(string numbers)
         {
-            return 0;
+            if (String.IsNullOrEmpty(numbers))
```
Comment (mikepiehler) [inline src/StringCalc/StringCalc.Core/StringCalculator.cs]: what about white space?

File: src/StringCalc/StringCalc.Core/StringCalculator.cs [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+            {
+                return 0;
+            }
+            string[] numberList = numbers.Split(',');
+            var addedNumber = 0;
+            foreach (string num in numberList)
```
Comment (mikepiehler) [inline src/StringCalc/StringCalc.Core/StringCalculator.cs]: why no linq?

**Comment by mikepiehler** [comment]
:watch:

File: src/StringCalc/StringCalc.Core/StringCalculator.cs (hunk 2 above)
Comment (THume) [inline src/StringCalc/StringCalc.Core/StringCalculator.cs]: Fixed

File: src/StringCalc/StringCalc.Core/StringCalculator.cs (hunk 1 above)
Comment (THume) [inline src/StringCalc/StringCalc.Core/StringCalculator.cs]: New requirement.

**Comment by mikepiehler** [comment]
:shipit:

**Pull request merged**